modules/cloud-config-container/envoy-sni-dyn-fwd-proxy/README.md

```diff
@@ -0,0 +1,61 @@
+# Containerized Envoy as SNI dynamic forward proxy on Container Optimized OS
+
+This module manages a `cloud-config` configuration that starts a containerized [Envoy SNI Dynamic forward proxy]https://www.envoyproxy.io/docs/envoy/latest/configuration/listeners/network_filters/sni_dynamic_forward_proxy_filter) service on Container Optimized OS running on port 443.
```

Comment:
I think, that maybe adding two sentences describing how this works, when it is useful, and involved risks, might be very helpful here.

Reply:
as discussed, we will add them once the apigee factory is there.